tests/pytest/conftest.py

```diff
@@ -39,55 +33,180 @@ def app_request(rf):
 
 
 @pytest.fixture
-def mocked_analytics_module(mocker):
-    """
-    Fixture returns a helper function to mock the analytics module imported on another given module.
-    """
+def model_PemData():
+    data = PemData(id=999, text="-----BEGIN PUBLIC KEY-----\nPEM DATA\n-----END PUBLIC KEY-----\n", label="Test public key")
+    data.save()
```

Comment:
For each instance creation statement, use the [`Class.objects.create()`](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#create) helper instead of the class, which avoids needing the `instance.save()` call.

Reply:
Fixed in [fcd76cf](https://github.com/cal-itp/benefits/pull/788/commits/fcd76cfdfb0a4806194e1927f3392f1fa1fc2830)